autopts/ptsprojects/mynewt/gap.py

```diff
@@ -212,13 +212,13 @@ def test_cases(ptses):
                   generic_wid_hdl=gap_wid_hdl),
         ZTestCase("GAP", "GAP/BOND/BON/BV-01-C",
                   cmds=pre_conditions +
-                  [TestFunc(btp.gap_set_io_cap, IOCap.no_input_output)],
+                  [TestFunc(btp.gap_set_io_cap, IOCap.no_input_output),
+                   TestFunc(btp.gap_wait_for_sec_lvl_change, 1, post_wid=108)],
                   generic_wid_hdl=gap_wid_hdl),
         ZTestCase("GAP", "GAP/BOND/BON/BV-03-C",
                   cmds=pre_conditions +
                   [TestFunc(btp.gap_set_io_cap, IOCap.no_input_output),
-                   # give some time before returning wid 118 to allow pair to finish
-                   TestFunc(sleep, 1, start_wid=118)],
+                   TestFunc(btp.gap_wait_for_sec_lvl_change, 1, post_wid=108)],
```

Comment:
Can we no add this wait in wid 118?

Reply:
So you only want to use this in GAP/BOND/BON/BV-01-C?

Reply:
No, I mean in wid/gap.py `hdl_wid_118()`.

Reply:
Ah, so in wid, not as post_wid. IIRC that wasn't working, but I'll take a look

Reply:
We can, I pushed the change, I think I was trying to wait at the end of 108, not in 118